content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -65,11 +65,8 @@ option. Your cluster requirements may need a different configuration.
 
 1.  Create a kube-apiserver load balancer with a name that resolves to DNS.
 
-    - In a cloud environment you should place your control plane nodes behind a TCP
-      forwarding load balancer. This load balancer distributes traffic to all
-      healthy control plane nodes in its target list. The health check for
-      an apiserver is a TCP check on the port the kube-apiserver listens on
-      (default value `:6443`).
+    - In a cloud environment, use the load balancer infrastructure provided by the cloud provider.
+      In an environment without a built-in load balancer (i.e. bare metal), you can use a TCP forwarding proxy,such as HAProxy, however you need to take steps to make that load balancer highly available. Otherwise your load balancer becomes a single point of failure. The health check for an apiserver is a TCP check on the port the kube-apiserver listens on (default value `:6443`).
```

Comment:
hello @RA489. Thanks for the PR.
I reread the comments from issue #15284.
It seems as if there could be three bullet points instead of a single paragraph,  1) cloud provider 2) bare metal 3) health check, if it applies to both environments

- The bare metal sentence could be split into two or more sentences. Possible edit:
In an environment without a built-in load balancer, such as bare metal, you can use a TCP forwarding proxy. However, ...
- There is an item further below in the list that refers to HAProxy.